### PR TITLE
ARK UI

### PR DESCRIPTION
**Pull request by dakejahl**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -112,43 +164,54 @@ sudo apt install -y \
 		curl \
 		jq \
 		snap \
-  		snapd
+		snapd \
+		avahi-daemon
 
 if [ "$TARGET" = "jetson" ]; then
 	sudo -H pip3 install Jetson.GPIO
```
Comment (AlexKlimaj) [inline setup.sh]: `sudo apt-get install libssl-dev` is needed on jetpack 6 to build logloader

File: platform/jetson/services/mavlink-router.service
Comment (AlexKlimaj) [inline platform/jetson/services/mavlink-router.service:1]: `sys-devices-platform-3550000.xudc-gadget-net-usb0.device` is not present on jetpack 6. Not sure what it is now.

**Comment by dakejahl** [comment]
Alright give it a try on your end, it's all working for me

**Comment by AlexKlimaj** [comment]
It seems like this is always starting in AP mode now even though there are known networks.

**Comment by dakejahl** [comment]
Alright go ahead and try this on a fresh install, I verified everything is installing and starting properly. Remember to unplug micro usb and reboot after running **setup.sh**. If this looks good we can merge this and https://github.com/ARK-Electronics/ARK-UI/pull/1

**Review by AlexKlimaj — APPROVED** [review]

**Pull request merged**